rs/bitcoin/adapter/test_utils/src/rpc_json.rs

```diff
@@ -0,0 +1,305 @@
+//! JSON types for Bitcoin daemon RPC API.
+//!
+//! Code in this module is adapted from https://github.com/rust-bitcoin/rust-bitcoincore-rpc/.
+//! The original license is CC0.
+
+use bitcoin::{address::NetworkUnchecked, Address, Amount, BlockHash, Network, ScriptBuf, Txid};
+use serde::{Deserialize, Serialize};
+use std::fmt;
+use std::str::FromStr;
+
+type AddressUnchecked = Address<NetworkUnchecked>;
+
+// Used for createrawtransaction argument.
+#[derive(Serialize, Clone, PartialEq, Eq, Debug, Deserialize)]
```

Comment:
Nit: Can we put the `Serialize`, `Deserialize` always to the end of the `derive` and in the same order? It just makes this a bit easier to read.

Reply:
Good idea. Fixed